app/test_split_integer.py

```diff
@@ -2,20 +2,41 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 17
+    number_of_parts = 4
+    result = split_integer(value, number_of_parts)
```

Comment:
You could avoid creating the 'result' variable here and directly use the function call in the assert statement. This would make your code more concise and easier to read.